tests/e2e/features/artifacts/bridgePage/artifacts.feature

```diff
@@ -102,6 +102,9 @@ Feature: Withdraw
     Given I go to page "/bridge?network=era-goerli"
     When I click by text "Withdraw"
     Then Element with "text" "Bridge" should be "visible"
+    Then Element with "partial text" "Recent withdrawals" should be "visible"
```

Comment:
Is "Number of tx in progress" missed? https://allure.matterlabs.dev/project/17/test-cases/1597?treeId=30 And what about other steps, it is hart now to have a parallel between steps in the code and tests. Can you please align or group steps at least?

Reply:
To add check for "Recent withdraw" counter to Withdraw tc